action.yml

```diff
@@ -53,7 +53,6 @@ runs:
         skip-verification :  ${{ inputs.skip-verification }}
     - name: Merge/approve PR
       uses: actions/github-script@v6
-      if: github.event_name == 'pull_request' && (github.actor == 'dependabot[bot]' || inputs.skip-verification == 'true')
```

Comment:
I would remove `inputs.skip-verification == 'true'` only - should it work?

Reply:
@Eomm I don't think so, from my tests it's the `github.actor == 'dependabot[bot]'` check that's failing

Reply:
ok folks so here's the rationale, since there seem to be some questions in this regard:

- we want to display warnings when the action's execution is skipped
- if we skip the action execution in the yaml, we don't have an easy way to display warnings (unless we put the warnings in the yaml, which was the first solution @guilhermelimak went for and which I don't like much)
- the action is using an external action https://github.com/dependabot/fetch-metadata/, which IIRC assumes that the action is built run in a dependabot context
- so what we want to do, is keep skipping that action without thereby changing the conditions for its executions
- remove the conditions of execution from the invocation of our custom code so we can handle them in JS code and print warnings accordingly